src/SQLQueryStress/Form1.cs

```diff
@@ -621,7 +621,7 @@ private void WriteBenchmarkCsvText(TextWriter tw)
         private void Form1_Load(object sender, EventArgs e)
         {
             var elemHost = new System.Windows.Forms.Integration.ElementHost();
-            sqlControl1 = new SqlControl();
+            //sqlControl1 = new SqlControl();

```

Comment:
Remove the line, do not leave a commented code.

Reply:
@ErikEJ , I will remove that line.